src/components/Form.jsx

```diff
@@ -1,29 +1,30 @@
 /* eslint-disable dot-notation */
 import React, { useState } from 'react';
 import styles from 'assets/scss/form.module.scss';
-import { useDispatch, useSelector } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
 import { addBook } from 'redux/books/bookSlice';
 
 const Form = () => {
   const [book, setBook] = useState({
     title: '',
     author: '',
+    category: '',
   });
 
   const dispatch = useDispatch();
-  const bookList = useSelector((state) => state.books.bookList);
 
   const handleSubmit = (event) => {
     event.preventDefault();
     const newBook = {
-      item_id: bookList.length + 1,
-      category: 'Action',
+      item_id: uuidv4(),
       ...book,
     };
```

Comment:
While you did a good job working on the project requirements I noticed you disabled stylelint and eslint in some files. It is against microverse policy to disable linters for linters help us write clean code, so it would be nice if you enabled eslint and stylelint in the following files

**Files** that have stylelint disabled
1. _global.scss
2. book.module.scss
3. categories.module.scss
4. form.module.scss
5. header.module.scss
6. styles.scss

Files that have eslint disabled 

1. Form.jsx
2. Book.jsx
3. pageheader.jsx
4. Categories.jsx
 Please kindly enable linters in all above files

Reply:
@KayLemba 

I have removed the eslint ignore as indicated above. However, I can't remove that of the scss files as I am using the @use to named import variables for use in other files. it is the recommended way to do so, and the style linter is flagging it. Also, the use of `color.$black` which is also the recommended way to reference the important modules, is being flagged again. Thus, the command is used to allow me to use those two approaches in my styling.